src/cmd/run.js

```diff
@@ -32,6 +32,7 @@ export type CmdRunParams = {|
   keepProfileChanges: boolean,
   noInput?: boolean,
   noReload: boolean,
+  reloadDelay: Number,
```

Comment:
Use the primitive `number` type instead of `Number`. Similarly in the rest of this PR.